const test = require('../../../..');

test('passes', t => {
	setTimeout(() => {
		throw new Error('Canâ€™t catch me');
	});
	t.pass();
});
